lib/screens/v2/widget_instance/train_crowding.ex

```diff
@@ -0,0 +1,82 @@
+defmodule Screens.V2.WidgetInstance.TrainCrowding do
+  @moduledoc """
+  A widget that displays the crowding on a train that is en route to the current station.
+  """
+
+  alias Screens.Config.Screen
+  alias Screens.Config.V2.Triptych
+  alias Screens.Predictions.Prediction
+
+  defstruct screen: nil,
+            prediction: nil,
+            now: nil
+
+  @type t :: %__MODULE__{
+          screen: Screen.t(),
+          prediction: Prediction.t(),
+          now: DateTime.t()
+        }
+
+  @type widget_data :: %{
+          destination: String.t(),
+          arrival_time: String.t(),
+          crowding: list(crowding_data),
+          platform_position: number,
+          front_car_direction: :left | :right,
+          now: String.t()
+        }
+
+  @type crowding_data :: %{
+          crowding_level: 1 | 2 | 3,
+          percentage: number
+        }
+
+  @spec serialize(t()) :: widget_data()
+  def serialize(%__MODULE__{
+        screen: %Screen{app_params: %Triptych{train_crowding: train_crowding}},
+        prediction: prediction,
+        now: now
+      }) do
+    %{
+      destination: prediction.trip.headsign,
+      arrival_time: serialize_time(prediction.arrival_time),
```

Comment:
Can this be omitted? I don't see us needing this in the client.

Reply:
Oh yes! ++ We did need it at one point when the logic was "if the train is 2 minutes away", but that's old now.